backend/LexBoxApi/Services/HgService.cs

```diff
@@ -271,8 +271,19 @@ private string[] GetWsList(System.Xml.XmlElement root, string tagName)
     {
         var result = await ExecuteHgCommandServerCommand(code, "flexmodelversion", token);
         var text = await result.ReadAsStringAsync(token);
-        var json = JsonDocument.Parse(text);
-        return json.RootElement.GetProperty("modelversion").GetInt32();
+        if (string.IsNullOrEmpty(text)) return null;
```

Comment:
I feel like this would be a minor improvement.
```suggestion
        if (string.IsNullOrWhitespace(text)) return null;
```

Reply:
Done in fd3d2174. I was thinking there's no need for that, but then I realized that if there ends up being a bug in the way the runner's result is returned through HTML, it's possible that we could end up with a `"\n"` string rather than an empty string.